inst/include/datarecord.h

```diff
@@ -46,7 +46,7 @@ class datarecord {
              int pos_, double id_);   
   
   //! short event constructor
-  datarecord(short int cmt_, int evid_, double amt_, double time_, double rate_);
+  datarecord(short int cmt_, int evid_, double amt_, double time_, double rate_, double fn_);
```

Comment:
This constructor is used when copying data record objects; adding `fn` as a reminder that this is a key item that needs to make it into the new object.